src/olympia/reviewers/templates/reviewers/addon_details_box.html

```diff
@@ -64,6 +64,15 @@ <h2>{{ addon_name }}</h2>
               </td>
             </tr>
           {% endif %}
+          {% if addon.support_url %}
+            <tr>
+              <th>{{ _('Support Website') }}</th>
+              <td>
+                <strong><a href="{{ addon.support_url }}">{{
```

Comment:
like the homepage link above, this url needs the `external_url` filter adding to it (it wraps the link in redirection service for safety)

Reply:
Sure